src/braket/aws/aws_quantum_task.py

```diff
@@ -446,11 +444,6 @@ def _(
     **kwargs,
 ) -> AwsQuantumTask:
     validate_circuit_and_shots(circuit, create_task_kwargs["shots"])
-    if circuit.qubits_frozen and not disable_qubit_rewiring:
-        raise ValueError(
-            "disable_qubit_rewiring must be True to run circuit with compiler directives"
-        )
```

Comment:
What error type does the service return? A change in the error type might be a breaking change here.

Reply:
I wasn't able to test that because we're outside the availability window (it just tells me the device is not available).

Reply:
On OQC? 
```python
ValidationException: An error occurred (ValidationException) when calling the CreateQuantumTask operation: Device arn:aws:braket:eu-west-2::device/qpu/oqc/Lucy does not support disabled qubit rewiring.
```

On Rigetti
```python
ValidationException: An error occurred (ValidationException) when calling the CreateQuantumTask operation: VerbatimBoxes cannot be used without disableQubitRewiring parameter set to "True".Please set disableQubitRewiring parameter to "True" or do not use VerbatimBoxes.
```